File: Master/Year 1/Semester 1/AMSS/Proiect/app/src/main/java/ro/robertto/rescuepets/presentation/fragment/LocationDetailsFragment.java
package ro.robertto.rescuepets.presentation.fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import ro.robertto.rescuepets.R;

public class LocationDetailsFragment extends Fragment {
    private TextView nameTextView;
    private TextView addressTextView;
    private TextView phoneTextView;

    @Nullable
    @Override
    public View onCreateView( @NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState ) {
        View view = inflater.inflate( R.layout.fragment_location_details, container, false );

        nameTextView = view.findViewById( R.id.nameTextView );
        addressTextView = view.findViewById( R.id.addressTextView );
        phoneTextView = view.findViewById( R.id.phoneTextView );

        // Get the location details from the arguments passed in
        Bundle arguments = getArguments();
        if ( arguments != null ) {
            String locationName = arguments.getString( "name" );
            String locationAddress = arguments.getString( "address" );
            String locationPhone = arguments.getString( "phone" );

            // Set the UI element values with the location details
            if ( nameTextView != null )
                nameTextView.setText( locationName );
            if ( addressTextView != null )
                addressTextView.setText( locationAddress );
            if ( phoneTextView != null )
                phoneTextView.setText( locationPhone );
        }

        return view;
    }
}

File: Bachelor/Anul 3/Semestrul 2/Android/Proiect/EmergencyApp/app/src/main/java/com/lifeSavers/emergencyapp/fragment/LocationDetailsFragment.kt
package com.lifeSavers.emergencyapp.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.Fragment
import com.lifeSavers.emergencyapp.R

class LocationDetailsFragment : Fragment() {
    private lateinit var nameTextView: TextView
    private lateinit var addressTextView: TextView
    private lateinit var phoneTextView: TextView

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val view = inflater.inflate(R.layout.fragment_location_details, container, false)

        nameTextView = view.findViewById(R.id.nameTextView)
        addressTextView = view.findViewById(R.id.addressTextView)
        phoneTextView = view.findViewById(R.id.phoneTextView)

        // Get the location details from the arguments passed in
        val locationName = arguments?.getString("name")
        val locationAddress = arguments?.getString("address")
        val locationPhone = arguments?.getString("phone")

        // Set the UI element values with the location details
        nameTextView.text = locationName
        addressTextView.text = locationAddress
        phoneTextView.text = locationPhone

        return view
    }
}
